test/unit/extension/lib/shopgate/CartExtensionPipelineTest.js

```diff
@@ -76,4 +76,8 @@ describe('CartExtensionPipeline - unit', () => {
 
     return subjectUnderTest.getCheckoutUrl().should.eventually.equal(expectedUrl)
   })
+  it('update products should return true', function () {
+    bigCommerceCartRepositoryMock.expects('updateItems').once().returns()
```

Comment:
I've pointed this out few times, I guess now you should see it - Add `withArgs` expectation. 
`returns` is not necessary.

Reply:
done